Add Sersync and Kmerfreq File (#273)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/kmerfreq.md

---
name: Kmerfreq
category: AI/ML

> description: kmerfreq is a tool for calculating the frequency of k-mers (subsequences of length k) in a sequence of biological data.

Author: ```suggestion
description: Kmerfreq is a tool for calculating the frequency of k-mers (subsequences of length k) in a sequence of biological data.
```

download_url: https://github.com/fanagislab/kmerfreq.git
works_on_arm: true
supported_minimum_version: 

>     version_number: master

Author: ```suggestion
    version_number: master branch
```

    release_date: 2019/08/21

optional_info:
    homepage_url: https://github.com/fanagislab/kmerfreq
    support_caveats:
    alternative_options: 
    getting_started_resources:
        arm_content: 
        partner_content: 
        official_docs: https://github.com/fanagislab/kmerfreq/blob/master/ReadMe.txt
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum: 
    release_notes__recommended_minimum:

>     other_info: There are no release notes available for ARM64. Kmerfreq was tested by cloning the repository and following the steps mentioned in the [README.md](https://github.com/fanagislab/kmerfreq/blob/master/ReadMe.txt).

Author: ```suggestion
    other_info: There are no releases in the Kmerfreq repository. Kmerfreq was tested by cloning the repository and following the steps mentioned in the [README.md](https://github.com/fanagislab/kmerfreq/blob/master/ReadMe.txt).
```
